src/schema.rs

```diff
@@ -309,6 +309,25 @@ mod schema_tests {
             "#),
             "my_int"
         ),
+        case::nullable_annotation_type_constraint(
+            load(r#"
+                    null
+                    null.null
+                    null.int
+                    0
+                    -5
+                "#),
+            load(r#"
+                    null.decimal
+                    a
+                    "hello"
+                    false
+                "#),
+            load_schema_from_text(r#" // For a schema with named type and `nullable` annotation as below: 
+                    type:: { name: my_int, type: nullable::int }
+                "#),
+            "my_int"
+        ),
```

Comment:
Does `ion-schema-tests` cover `nullable::` on the other types?

Reply:
No, only on built in types.

Reply:
That's what I meant, the non-`int` built in types. I wanted to make sure our tests covered `float`, `list`, etc.

Reply:
Oh okay. Sure I will add some tests here for `float`, `list`, `string` etc.